tests/test_pytypes.cpp

```diff
@@ -206,7 +206,11 @@ TEST_SUBMODULE(pytypes, m) {
     m.def("str_from_char_ssize_t", []() { return py::str{"red", (py::ssize_t) 3}; });
     m.def("str_from_char_size_t", []() { return py::str{"blue", (py::size_t) 4}; });
     m.def("str_from_string", []() { return py::str(std::string("baz")); });
+    m.def("str_from_string_input", [](const std::string &stri) { return py::str(stri); });
```

Comment:
I'd probably make this `str_from_std_string_input` because `string` is so easily misunderstood (in the call site).

Reply:
I don't think it's critical, it's not too hard to infer. It is slightly better though.

I think this is fine for 2.10.1.

Reply:
```suggestion
    m.def("str_from_std_string_input", [](const std::string &stri) { return py::str(stri); });
```

Reply:
I just wanted to be consistent with the out pytest unit tests, but yeah this change is fine.